### DISCUSSION TITLE
does skywlking OAP planning to support 'Open-Telemetry' data structure ?

### DISCUSSION THREAD

**Discussion by eueuy** [apache/skywalking#7684] [Q&A] [answered]
we use many kind of tracing-agent to collect span from programs write by diffrent lang .  for java ,its skywalking .   now we happy to see an open-standard come out .  so that can help us to remove adapter service in our arch.   does skywlking planning to support Open-Telemetry?

**Comment by wu-sheng** [comment]
Please confuse readers. Opentelemetry is not ever closing a standard. 
Don't use this term.

**Comment by wu-sheng** [comment, marked as answer]
We have much more powerful agents with our native and more APM focused design. For tracing, this has been explained by STAM paper. 
The oc format of OpenTelemetry metrics exporter has been supported for months.
The log format could be supported easily as we supported filebeat, fluentd/fluentbit, and native SkyWalking log format.

So, we are doing meaningful things, contributed by the whole community.
Don't waste the time and make things worse.
